puppet/modules/site_check_mk/manifests/agent/tapicero.pp

```diff
@@ -17,10 +16,5 @@
         'rm /files/etc/check_mk/mrpe.cfg/Tapicero_Procs',
         'set Tapicero_Procs "/usr/lib/nagios/plugins/check_procs -w 1:1 -c 1:1 -a tapicero"' ],
       require => File['/etc/check_mk/mrpe.cfg'];
-    'Tapicero_Heartbeat':
-      incl    => '/etc/check_mk/mrpe.cfg',
-      lens    => 'Spacevars.lns',
-      changes => 'set Tapicero_Heartbeat \'/usr/local/lib/nagios/plugins/check_last_regex_in_log -f /var/log/syslog -r "tapicero" -w 300 -c 600\'',
-      require => File['/etc/check_mk/mrpe.cfg'];
   }
```

Comment:
unless the tests are running frequently, it is not a bug for there to be no tapicero activity.